lms/djangoapps/courseware/tests/test_split_module.py

```diff
@@ -131,11 +131,11 @@ def _check_split_test(self, user_tag):
             u'<button class="{} inactive nav-item tab"'.format(self.ICON_CLASSES[user_tag]),
             content.decode(resp.charset)
         )
+        unicode_content = content.decode("utf-8")
```

Comment:
Why not use resp.charset like the assert above this?  Maybe pull up unicode content above that assert and use the unicode content for all 3  assertions?